completion.go

```diff
@@ -68,7 +68,7 @@ const (
 	}
 
 	__hcloud_image_types_no_system() {
-		COMPREPLY="snapshot backup"
```

Comment:
I think your previous code would work, but you missed the parens: `COMPREPLY=("snapshot" "backup")`

Reply:
does not work :(